modyn/protos/model_storage.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+package modyn.model_storage;
+
+service ModelStorage {
+  rpc RegisterModel(RegisterModelRequest) returns (RegisterModelResponse) {}
+  rpc FetchModel(FetchModelRequest) returns (FetchModelResponse) {}
+  rpc DeleteModel(DeleteModelRequest) returns (DeleteModelResponse) {}
+}
+
+message RegisterModelRequest {
+  int32 pipeline_id = 1;
+  int32 trigger_id = 2;
+  string hostname = 3;
+  int32 port = 4;
+  string model_path = 5;
+}
+
+message RegisterModelResponse {
+  bool success = 1;
+  int32 model_id = 2;
+}
+
+message FetchModelRequest {
+  int32 model_id = 1;
+}
+
+message FetchModelResponse {
+  bool success = 1;
+  string model_path = 2;
+}
+
+message DeleteModelRequest {
+  int32 model_id = 1;
+}
+
+message DeleteModelResponse {
+  bool valid = 1;
```

Comment:
maybe call this success instead of valid?

Reply:
I agree, done!